mozilla-release/toolkit/components/url-classifier/SafeBrowsing.jsm

```diff
@@ -191,19 +191,19 @@ this.SafeBrowsing = {
       case "Phish":
         pref = "browser.safebrowsing.reportPhishURL";
         Services.telemetry.getHistogramById("REPORT_DECEPTIVE_SITE").add(0);
-        reportPhishingURL(URI.asciiSpec, 'user-report-phish');
+        reportPhishingURL(info.uri, 'user-report-phish');
```

Comment:
are you sure this is correct? `asciiSpec` is a string, while URIs are typically objects..

Reply:
I checked in debugger - "uri" is string in this "info" object.